stable/traefik/templates/_helpers.tpl

```diff
@@ -7,3 +7,7 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 {{- define "fullname" -}}
 {{- printf "%s-%s" .Release.Name .Chart.Name | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
+
+{{- define "dashboardUsers" -}}
+{{- range $key, $value := .Values.dashboard.auth.basic -}}{{- printf "%s:%s\n" $key $value -}}{{- end -}}
```

Comment:
Not really. The only thing that calls this is wrapped in those conditionals already and I don't foresee this being reused in this chart. (I didn't do it this way for the sake of reuse... this "sub-template" was the only way I could think of, at the time, to build a string by ranging over an array, and then get that string into a pipeline so I can base64 encode it... but I think I've thought of a more straightforward way to do that now, so I'll be updating this on Monday.)